.github/workflows/e2e-tests-wp-nightly.yml

```diff
@@ -1,11 +1,48 @@
 name: E2E Tests
 
+defaults:
+  timeout-minutes: 20
+
 on:
   push:
     branches:
       - develop
       - main
+    paths-ignore:
+      # Ignore Storybook-only and unit-test-only changes when checking to run
+      # E2E tests. This is only useful when
+      # adding/modifying/refactoring/removing test code ONLY in a commit, but
+      # saves running the expensive E2E test suite on GitHub Actions when
+      # all that's changing are tests and not code that would affect E2E
+      # behavior.
+      - '.storybook/**'
+      - 'stories/**'
+      - 'tests/backstop/**'
+      - 'tests/phpunit/**'
+      - 'assets/**/__tests__/**/*.js'
+      - 'assets/**/test/*.js'
+      - 'assets/**/?(*.)test.js'
+      - 'packages/**/__tests__/**/*.js'
+      - 'packages/**/test/*.js'
+      - 'packages/**/?(*.)test.js'
```

Comment:
I don't think this will work as expected because filter patterns aren't regexs.

See the docs here: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
I saw that but thought it would work, but looking again I think you're right. Okay, I'll tweak! 👍🏻